CarRentals/CarRentals/Controllers/Crud.cs

```diff
@@ -0,0 +1,50 @@
+﻿using CarRentals.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CarRentals.Controllers
+{
+    public abstract class Crud<T> : IDataProcessing<T> where T : IModelForCrud
+    {
+        protected List<T> List { get; set; }
+        public ManageJson<T> Json { get; set; }
```

Comment:
This property name is particularly odd since it doesn't really matter which approach you use. Yes, you started with a `JSON` as a way to storage data, but tomorrow can be a REDIS database. Maybe something generic like... `Storage`?

Reply:
But if you follow my advice above... this property will not exist anymore since this class could handle all the json stuff.